Site/SiteJwPlayerMediaDisplay.php

```diff
@@ -323,7 +329,7 @@ protected function getJavascript()
 			$javascript.= sprintf("\t%s.addSource(%s, %d, %s);\n",
 				$this->getJavascriptVariableName(),
 				SwatString::quoteJavaScriptString($source['uri']),
-				$source['width'],
+				($source['width'] === null) ? "''" : $source['width'],
```

Comment:
Should this be comparison be `!= ''`? We're defaulting to an empty string on line 173.